packages/lodestar-beacon-state-transition/src/fast/signatureSets/index.ts

```diff
@@ -0,0 +1,31 @@
+import {BeaconState, SignedBeaconBlock} from "@chainsafe/lodestar-types";
+import {getRandaoRevealSignatureSet} from "../block/processRandao";
+import {getBlockSignatureSet} from "../util/block";
+import {EpochContext} from "../index";
+import {ISignatureSet} from "./types";
+import {getProposerSlashingsSignatureSets} from "./proposerSlashings";
+import {getAttesterSlashingsSignatureSets} from "./attesterSlashings";
+import {getAttestationsSignatureSets} from "./attestations";
+import {getVoluntaryExitsSignatureSets} from "./voluntaryExits";
+
+export * from "./types";
+export * from "./verify";
+
+/**
+ * Includes all signatures on the block (except the deposit signatures) for verification.
+ * Deposits are not included because they can legally have invalid signatures.
+ */
+export function getAllBlockSignatureSets(
```

Comment:
cannot find where do we use this. I assume in processBlock and if it passes all verifySignatures should be set to false?

Reply:
This is not used anywhere yet but should be used as a pre-step to running the state-transition.